supabase/functions/process-job-queue/index.py

```
@@ -0,0 +1,125 @@
+import os
+import supabase
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)